### ISSUE TITLE
New weapon: CryShard

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#253]
Very fragile and totally crying about it.

copy from HeavySniper
damage 20

Special: during continuous fire, this weapon's firerate increases. After 4 shots it reaches 260ms and the weapon switches modes: it is now a shotgun, no longer gains firerate, and fires 2 pellets. Next shot it fires 3 pellets. Next shot it fires 4 and so on, until it finally consumes the entire magazine. (Though, perhaps it would be funnier if this weapon was belt-fed?) Spread also increases with each shot.

The damage per shot always remains the same, and is equally divided by all the pellets.

**Comment by ghost** [comment]
So what exactly is the mag size of this weapon? If a copy of HS, then it only has 6 shots, which means it can only fire 2 shotgun shots, and takes up more than half of its mag size before the firerate increases. 
I would recommend either increasing mag size, or making the changes occur after the 3rd or 2nd shot.
Also, I'll assume when that by "during continuous fire", you mean holding down your attack key, and not releasing it, until a certain amount of bullets are automatically fired from the weapon, and not if someone simply shoots 4 bullets from their mag.

**Comment by deferred-impact** [comment]
Yes, continuous fire means continuous fire. Letting go of the trigger will rapidly reset the weapon.

The mag size is definitely up to discussion. 6 is probably not enough, yes. There's also the option of just not having a magazine at all, which may be applicable.

**Comment by ghost** [comment]
I'd say around 10 - 12 shots would be fine.
